week10/MovieViewer/app/src/main/java/com/wmw/movieviewer/ui/MovieActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.wmw.movieviewer.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.Observer
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.wmw.movieviewer.R
+import com.wmw.movieviewer.model.Movie
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.coroutines.launch
+import org.koin.android.ext.android.inject
+import org.koin.core.KoinComponent
+
+fun startMainActivity(from: Context) = from.startActivity(Intent(from, MovieActivity::class.java))
+
+class MovieActivity : AppCompatActivity(), KoinComponent {
```

Comment:
You don't need to add KoinComponent everywhere... Activities and Fragments have the inject and ViewModel functions by default.